modules/cnv-cloning-local-volume-to-another-node.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-moving-local-vm-disk-to-different-node.adoc
+
+[id="cnv-cloning-local-volume-to-another-node_{context}"]
+= Cloning a local volume to another node
+
+You can move a virtual machine disk so that it runs on a specific node by cloning the underlying PersistentVolumeClaim (PVC).
+
+To ensure the virtual machine disk is cloned to the correct node, either create a PersistentVolume (PV) or identify one already on the node and apply a unique label.
+Reference this label in the DataVolume so that the underlying PVC of the virtual machine is cloned to the PV on the correct node. 
+
+[NOTE]
+====
+The destination PV must be the same size or larger than the source PVC or else the cloning operation will fail.
+====
+
+.Prerequisites
+
+* The virtual machine must not be running. Power down the virtual machine before cloning the virtual machine disk. 
+
+.Procedure
+
+. Either create a new local PV on the node, or identify a local PV already on the node:
+
+* Create a local PV that includes the `nodeAffinity.nodeSelectorTerms` parameters. The following manifest creates a 10Gb local PV on node01. 
++
+[source,yaml]
+----
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: <destination-pv> <1>
+  annotations:
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 10Gi <2>
+  local:
+    path: /mnt/local-storage/local/disk1 <3>
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - <node01> <4>
+  persistentVolumeReclaimPolicy: Delete
+  storageClassName: local
+  volumeMode: Filesystem
+----
+<1> The name of the PV.
+<2> The size of the PV. You must allocate enough space, or the cloning operation fails. The size must be the same as or larger than the source PVC.
+<3> The mount path on the node.
+<4> The name of the node on which you want the PV to be created.
+
+* You can identify the node on which the PV is provisioned by examining the `nodeAffinity` field of the PV configuration, in particular the `kubernetes.io/hostname` key and matching values:
++
+----
+$ oc get pv <destination-pv> -o yaml
+----
++
+The following snippet shows that the PV is on `node01`:
++
+[source,yaml]
+----
+...
+spec:
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node01
+...
+----
+
+. Once you have created or identified the PV on the correct node, add a unique label to that PV:
++
+----
+$ oc label pv <destination-pv> node=node01
+----
+
+. Create a DataVolume manifest that references the PVC of the virtual machine you are cloning to the new node, and the `selector` label that was applied to the PV in the previous step:
```

Comment:
I don't think this should say `selector` because you didn't mention `selector` in the previous step. I would also rather this be more minimalistic. 

Suggestion:
> Create a DataVolume manifest that references:
> - The PVC of the virtual machine that you want to clone.
> - The label that you applied to the PV in the previous step.
> For example: